test/functional/segwit.py

```diff
@@ -70,7 +70,7 @@ def getutxo(txid):
 
 def find_unspent(node, min_value):
```

Comment:
nit: Rename to `find_spendable_output` or something.

Reply:
That would make the PR much more disruptive, is it worth it?
Also, if I add the require_spendable param, I don't think it makes sense to rename it to that.

Reply:
> much more disruptive.

It is used in less than 5 places.

Changing the function name when the functionality changes aims review and prevents silent merge conflicts/bugs.